PirateGame/Assets/Scripts/GUIFunctions.cs

```diff
@@ -4,18 +4,26 @@
 public class GUIFunctions : MonoBehaviour {
 	public Ship ship;
 
+	//change from public to private and set values by getting object with correct tag from GameObject.Find("tag_here");
```

Comment:
Right, that's what I'm trying to avoid having to do every time.  We should never have to do manual setup to get the game to run, it should all be automated.  I will bring this up at today's meeting so we can discuss.